src/Task/Deptrac.php

```diff
@@ -21,25 +21,13 @@ public static function getConfigurableOptions(): OptionsResolver
         $resolver = new OptionsResolver();
         $resolver->setDefaults([
             'depfile' => null,
-            'formatter' => [],
-            'graphviz_display' => true,
-            'graphviz_dump_image' => null,
-            'graphviz_dump_dot' => null,
-            'graphviz_dump_html' => null,
-            'junit_dump_xml' => null,
-            'xml_dump' => null,
-            'baseline_dump' => null,
+            'formatter' => null,
+            'output' => null,
         ]);
 
         $resolver->addAllowedTypes('depfile', ['null', 'string']);
-        $resolver->addAllowedTypes('formatter', ['string[]']);
-        $resolver->addAllowedTypes('graphviz_display', ['bool']);
-        $resolver->addAllowedTypes('graphviz_dump_image', ['null', 'string']);
-        $resolver->addAllowedTypes('graphviz_dump_dot', ['null', 'string']);
-        $resolver->addAllowedTypes('graphviz_dump_html', ['null', 'string']);
-        $resolver->addAllowedTypes('junit_dump_xml', ['null', 'string']);
-        $resolver->addAllowedTypes('xml_dump', ['null', 'string']);
-        $resolver->addAllowedTypes('baseline_dump', ['null', 'string']);
+        $resolver->addAllowedTypes('formatter', ['null', 'string']);
```

Comment:
Shouldnt this be `null|string[]`, since you can use multiple formatters?
https://github.com/qossmic/deptrac/blob/07bc3f50449b751245d4f83c683de5250be15d60/src/Console/Command/AnalyseCommand.php#L63-L70

Reply:
No, because `formatter` no longer allows to use multiple values (`InputOption::VALUE_IS_ARRAY | InputOption::VALUE_REQUIRED` was changed by `InputOption::VALUE_OPTIONAL` in 0.19).